docs/guide/variadics/README.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Variadic Nodes
+---
+
+# Variadic Nodes
+
+There are operations which can be applied to an arbitrary number of input
+values. For example, consider the addition operation. Sure, intuitively you
+know how it works for two values, three values, or 42 values:
+
+* 1 + 6 = 7
+* 1 + 6 + 3 = 10
+* 1 + 6 + 3 + …more summands… = someNumber
+
+Other examples of such functions include number multiplication, string
+concatenation or joining, booleans AND’ing or OR’ing, n-th element choice, and
+so on.
+
+To natively support such operations, XOD offers a mechanism called *variadic*
+nodes. The variadic nodes can scale the number of their input pins to accept
+more or fewer values.
+
+In XOD IDE the variadic nodes have a little gripper on the right, and the pins
+which can scale are marked with the ellipsis (⋯) on mouse hover. Depending on
+requirements you can add groups of variadic pins up to an infinite number. Drag
+the right border of the node to adjust.
+
+![Overview](./overview.patch.png)
+
+The variadic pins replication count is called *arity level*. By default, the
+arity level for a node is 1. When you add a pin to, for example, the `multiply`
+node, the arity level becomes 2, etc.
+
+## Expansion
+
+A variadic node developer is responsible only for creating a node
+implementation for the first arity level. All subsequent arity level
+implementations are induced automatically by XOD. To achieve it, when
+transpiling the program each particular variadic node passes an *expansion*
+stage in which the system replaces it with a cascade of primitive level-1
+nodes. The replacement principle is better illustrated with an example.
+
+![Expansion](./expansion.patch.png)
+
+<div class="ui segment note">
+<span class="ui ribbon label">Pro</span>
+Experienced developers with functional programming experience could note the
```

Comment:
"**Experienced** developers with functional programming **experience**"

Reply:
Savvy developers with functional programming experience?

Reply:
I vote for "Developers with functional programming experience" 🙂